src/main/java/me/moodcat/database/entities/User.java

```diff
@@ -0,0 +1,67 @@
+package me.moodcat.database.entities;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * User entity.
+ */
+@Data
+@Entity
+@Table(name = "user")
+@EqualsAndHashCode(of = {
+	"id"
+})
+public class User {
+
+	/**
+	 * The unique id of the user.
+	 *
+	 * @param id
+	 *            The new Id to set.
+	 * @return The id of the user.
+	 */
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "id", nullable = false)
+	private Integer id;
+
+	/**
+	 * The unique id of the user.
+	 *
+	 * @param id
+	 *            The new Id to set.
+	 * @return The id of the user.
+	 */
+	@Column(name = "soundcloud_id", nullable = true, unique = true)
+	private Integer soundcloud_id;
```

Comment:
Since we use soundCloudId for songs (camelcase, this has _), maybe we should rename this to soundCloudUserId